src/main/java/me/tongfei/progressbar/ProgressState.java

```diff
@@ -15,11 +17,16 @@
     long max = 0;
     Instant startTime = null;
     String extraMessage = "";
+    long start = 0;
+    Duration elapsed = Duration.ofSeconds(0L);
 
-    ProgressState(String task, long initialMax) {
+    ProgressState(String task, long initialMax, long startFrom, long elapsedSecond) {
```

Comment:
the argument `long elapsedSecond` would be better changed to `Duration elapsed`.